### DISCUSSION TITLE
.NET Kubernetes Operators

### DISCUSSION THREAD

**Discussion by buehler** [dotnet/dotnet-operator-sdk#191] [Ideas & General Discussion]
Copied from Issue #190.

from @sebagomez:
Hey, you're hard to find so this is the only way I found.

I think you have a great thing here, as the matter of fact, I have kind of the same :)
I'm the maintainer of a similar project where I want to have C# (or .NET) developers easily write Kubernetes Operators. You can take a look at my project here.

I was wondering if you wanted to join for a quick chat and discuss the future of .NET in the Kubernetes Operators world.

You can reply this issue and/or reach out directly to me by email.
Cheers

**Comment by buehler** [comment]
Alright @sebagomez 
How would you like to discuss the matter? Should we do it here asynchronously or would you like to have some kind of meeting? I could offer Google Meet if you'd like.

**Reply by sebagomez** [reply]
I would prefer a first meeting if that's ok with you. 
If you're based in Zurich it would need to be my morning so it's your afternoon, having said that, this week's mornings are packed, can we arrange something for next week?

Here's my email so we can discuss times and share the meet's link: I'm Sebastian dot Gomez at container-solutions.com

Cheers!

**Reply by buehler** [reply]
Yea that's totally fine :-)

**Reply by tomkerkhove** [reply]
May I ask what the outcome was?

**Reply by buehler** [reply]
Oh, of course! :-)
We met and talked about the project and the motivation behind our projects in general.

@sebagomez will take KubeOps and the basic idea of to his team and they will discuss if they want to support this project as contributors or not.

Since I'm doing this in my free time without any company backup, development will stall from time to time.

**Reply by tomkerkhove** [reply]
Any update on this @sebagomez? Also, are you open to more contributions @buehler?

**Reply by sebagomez** [reply]
Hey, I have not had the time to work on it, sorry. 
I will block some time for it though.

**Reply by tomkerkhove** [reply]
I presume the current state is the same - Is that correct?
